tests/ops/qubit/test_parametric_ops.py

```diff
@@ -1001,44 +1004,56 @@ def get_expected(theta):
             qml.PSWAP(param, wires=[0, 1]).matrix(), get_expected(param), atol=tol, rtol=0
         )
 
-    @pytest.mark.parametrize("phi", np.linspace(-np.pi, np.pi, 10))
+    @pytest.mark.parametrize("phi", pswap_angles)
```

Comment:
`pswap_angles` differs from the previous angles by adding the `0` angle to the test set, and by appending a broadcasting case to the end of the set of parameters.